lib/person.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+require 'pry'
+
+class Person
+  attr_accessor :portfolio
+
+  def initialize
+    @portfolio = []
+  end
+
+  def rent(library, args={})
+    #binding.pry
+    item = library.books.detect { |obj| obj[args.keys.first] == args[args.keys.first] }
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)